cnd/src/http_api/mod.rs

```diff
@@ -503,10 +504,10 @@ mod tests {
     #[test]
     fn http_asset_serializes_correctly_to_json() {
         let bitcoin = HttpAsset::from(asset::Bitcoin::from_sat(100_000_000));
-        let ether = HttpAsset::from(asset::Ether::from_eth(1.0));
+        let ether = HttpAsset::from(asset::Ether::from_wei(1_000_000_000_000_000_000u64));
```

Comment:
Can we have a cost or comment that indicates that this is one Ether in Wei please, I would not have known this without the diff.